docs/ci.md

```diff
@@ -0,0 +1,32 @@
+---
+id: ci
+title: Run Prettier on CI
```

Comment:
I prefer "GitHub Actions" instead of "CI", because CI is not only GA.

```suggestion
title: Run Prettier on GitHub Actions
```

Reply:
Great, I think it would be better the title is `Run Prettier on CI` and there is sub-section `GitHub Actions`.